draft-ietf-quic-transport.md

```diff
@@ -1168,35 +1168,86 @@ Destination and Source Connection ID fields match the Source and Destination
 Connection ID fields in a packet that the client sent.  If this check fails, the
 packet MUST be discarded.
 
+A client MUST ignore a Version Negotiation packet that lists the client's chosen
```

Comment:
```suggestion
A client MUST discard a Version Negotiation packet that lists the client's chosen
```
This is a minor nit.  Let's keep the wording consistent.  Word "discard" was used in a sentence above.